editors/vscode/README.md

```diff
@@ -5,14 +5,16 @@ This extension provides support for the [Lambdapi](https://github.com/Deducteam/
 Goals are visualised in a panel on the right side of the editor.
 You can navigate in proof with the following key-bindings:
 
-- ``Ctrl+Right``: go one step forward
-- ``Ctrl+Left``: go one step backward
-- ``Ctrl+Up``: go to the previous proof (or the beginning)
-- ``Ctrl+Down``: go to the next proof (or the end)
+- ``Ctrl+Right`` (``Ctrl+fn+Right`` in `Mac OS X`): go one step forward
```

Comment:
Why not structure in 2 different sections: one for Linux and one for Mac OS X (duplication does not matter in this case)?